### ISSUE TITLE
Tooltip:  [aria-*] attributes are not valid or misspelled

### ISSUE THREAD

**Issue by tvld** [bootstrap-vue/bootstrap-vue#3310]
Google Lighthouse gives `` [aria-*] attributes are not valid or misspelled`` Failing Elements on: 
```html
<svg role="img" aria-hidden="false" width="16" height="16" viewBox="0 0 512 512" focusable="false" aria-labelled-by="vat-va-d493e" class="fa-icon tool-tip-li" data-original-title="" title="">
```
code:
```html
  <span v-b-tooltip.hover
          class="tool-tip-li"
          title="My tooltip"
          v-html = "'Hello'">
```

Version:  ``"bootstrap-vue": "^2.0.0-rc.19"``

**Comment by tmorehouse** [comment]
This might be related to placing the tooltip directly on an SVG element (which is not focusable).

If you are placing the tooltip on a wrapper span, it should be placing the aria-attributes on the span, and not the SVG.

**Comment by jackmu95** [comment]
@tvld @tmorehouse `aria-labelledby` is misspelled in the SVG markup above (`aria-labelled-by`).

**Comment by tmorehouse** [comment]
We are only setting `aria-describedby` in tooltips and popovers, not `aria-labeledby`

**Comment by tmorehouse** [comment]
@tvld it looks like the issue is not with tooltips, but with the markup in your SVG element.

**Comment by tvld** [comment]
That's weird. the icon is from ``vue-awesome``; I have this as link:

```js
<router-link :to="`/blog/${post.nr}`" v-b-tooltip.hover :title="$t('resources.blog.share')">
   <icon style="cursor:pointer;" name="share-alt" />
   #{{ post.nr }}
</router-link>
```

**Comment by tmorehouse** [comment]
The issue is with vue-awesome Icon component:

https://github.com/Justineo/vue-awesome/blob/master/src/components/Icon.vue#L206-L208

 ```js
    let titleId = `vat-${this.id}`
    if (this.title) {
      options.attrs['aria-labelled-by'] = titleId
    }
```

I would suggest opening an issue or PR at https://github.com/Justineo/vue-awesome

**Comment by tvld** [comment]
Excellent find. I will open the issue. Thanks ! ))

**Comment by tmorehouse** [comment]
Created a PR for it https://github.com/Justineo/vue-awesome/pull/147
